tools/phenomenal/isatools/create_metabo/isa_create_metabo.xml

```diff
@@ -7,21 +7,22 @@
 
     <requirements>
         <requirement type="package" version="6.7">click</requirement>
-        <requirement type="package" version="0.9.4">isatools</requirement>
+        <!--<requirement type="package" version="0.9.4">isatools</requirement>-->
     </requirements>
 
     <stdio>
         <exit_code range="1:" />
     </stdio>
-
     <command>
         <![CDATA[
-        cli.py
 
-        --galaxy_parameters_file='$inputs'
-        --target_dir='./' &&
+        mkdir study_dir &&
         mkdir '${output.files_path}' &&
-        cp ./*.txt '${output.files_path}'
+        $__tool_directory__/cli.py
+        --galaxy_parameters_file='$inputs'
+        --target_dir='study_dir/' &&
+        cp study_dir/* '${output.files_path}'
```

Comment:
I would suggest a `mv study_dir/* '${ouput.files_path}'` instead, avoid the copy, unless needed.

Reply:
Ahhh, I see why you need the cp downwards, because you are collecting files from study_dir...

Reply:
Yeah, the `output.files_path` is for the `isa-tab` datatype to pick up, and then the other directory is for the output collections.